api/app/controllers/transcript.controller.ts

```diff
@@ -52,12 +51,33 @@ export async function create(req: Request, res: Response) {
     contentTotalWords: totalWords,
   };
 
-  // Save Transcript in the database
   try {
     const transcriptData = await Transcript.create(transcript);
+    const redisTransaction = redis.multi();
```

Comment:
isn't `redisTransaction` misleading when the function is to save transcripts?

Reply:
What do you suggest? I named it so because its a redis transaction.

Reply:
something more explicit like `redisNewTranscriptTransaction`
or a comment above, just because we have the concept of transaction in the codebase and this may be confusing